### PR TITLE
#BAH-972 | Vinay | Import of Forms 2.0 in Implementer Interface

### PR DESCRIPTION
**Pull request by vinay-ThoughtWorks**

### CODE REVIEW

File: src/form-builder/components/FormBuilder.jsx [hunk 1]
```diff
@@ -68,6 +76,66 @@ export default class FormBuilder extends Component {
...
+      } else {
+        this.updateImportErrors(fileName, 'Invalid file format');
+      }
+    });
+    return validJsonFiles;
+  }
+
+  validateAndLoadZipFile(jsonZip) {
+    const self = this;
+    const maxAllowedSize = 500 * 1024;
```
Comment (bsneha90) [inline src/form-builder/components/FormBuilder.jsx]: can we make this configurable? or larger? I see today most of our forms are larger that 500KB. So we are unable to import a single form also.

File: src/form-builder/components/FormBuilder.jsx [hunk 2]
```diff
@@ -85,75 +153,165 @@ export default class FormBuilder extends Component {
...
+      }
+    });
+    self.processFormValidationPromises(formsValidationPromises);
+  }
+
+  importJsonFile(file) {
+    const self = this;
+    const reader = new FileReader();
+    const fileName = file[0].name;
     // eslint-disable-next-line
```
Comment (bsneha90) [inline src/form-builder/components/FormBuilder.jsx]: is the  eslint ignore valid for this line? // eslint-disable-next-line

File: src/form-builder/components/FormBuilder.jsx (hunk 1 above)
Comment (vinay-ThoughtWorks) [reply src/form-builder/components/FormBuilder.jsx]: Increased to 5MB

File: src/form-builder/components/FormBuilder.jsx (hunk 2 above)
Comment (vinay-ThoughtWorks) [reply src/form-builder/components/FormBuilder.jsx]: yes

File: src/form-builder/components/FormBuilder.jsx (hunk 1 above)
Comment (vinay-ThoughtWorks) [reply src/form-builder/components/FormBuilder.jsx]: Done. Increased to 5MB

File: src/form-builder/components/FormBuilder.jsx (hunk 2 above)
Comment (vinay-ThoughtWorks) [reply src/form-builder/components/FormBuilder.jsx]: Yes

**Pull request merged**